### PR TITLE
Feat release 1.5

### PR DESCRIPTION
**Pull request by juzhiyuan**
related PR https://github.com/apache/apisix-dashboard/pull/356

### CODE REVIEW

**Comment by juzhiyuan** [comment]
@gxthrj Please take a look at those comments: https://github.com/apache/apisix-dashboard/pull/356

File: README.md [hunk 1]
```diff
@@ -36,28 +36,29 @@ The `manager-api` is used to provide APIs for Dashboard, just like a bridge betw
 ```sh
 # e.g Initialization for MySQL
 $ mysql –uroot –p123456
-> source apisix-dashboard/api/script/db/schema.sql
+> source $PROJECT_ROOT_FOLDER/api/script/db/schema.sql
```
Comment (juzhiyuan) [inline README.md:39]: Please use `apisix-dashboard` instead.

File: README.md [hunk 2]
```diff
@@ -36,28 +36,29 @@ The `manager-api` is used to provide APIs for Dashboard, just like a bridge betw
...
+3. Check environment variables
 
-According to your local deployment environment, modify the environment variables in `$PROJECT_ROOT_FOLDER/api/run/run.sh`
+According to your local deployment environment, check the environment variables in `$PROJECT_ROOT_FOLDER/api/run/run.sh`, modify the environment variables if necessary.
 
 4. Build
 
 ```sh
-$ go build -o manager-api apisix-dashboard/api
+## make sure you are in the path of $PROJECT_ROOT_FOLDER
```
Comment (juzhiyuan) [inline README.md:53]: @moonming recommends we should use `apisix-dashboard` here

File: README.md [hunk 3]
```diff
@@ -36,28 +36,29 @@ The `manager-api` is used to provide APIs for Dashboard, just like a bridge betw
...
+$ go build -o manager-api ./api
 ```
 
 5. Run
 
 ```sh
-$ chmod +x apisix-dashboard/api/run/run.sh
-$ sh apisix-dashboard/api/run/run.sh
+## make sure you are in the path of $PROJECT_ROOT_FOLDER
```
Comment (juzhiyuan) [inline README.md:60]: same issue here

File: README.md (hunk 1 above)
Comment (gxthrj) [reply README.md:39]: I need absolute path here

File: README.md (hunk 2 above)
Comment (gxthrj) [reply README.md:53]: this is relation path, when run manager-api locally, need to  build manager-api in the root path of apisix-dashboard

File: README.md (hunk 3 above)
Comment (gxthrj) [reply README.md:60]: this is relation path, when run manager-api locally, need to run `run.sh` in the root path of apisix-dashboard

**Pull request merged**